content/1_docs/2_cookbook/9_setup/0_selective-page-cache/recipe.txt

```diff
@@ -0,0 +1,117 @@
+Title: Disabling the page cache for logged-in users
+
+----
+
+Published: 2020-07-09
+
+----
+
+Description: Customize pages for authenticated users while using the default page cache.
+
+----
+
+Text:
+
+Using Kirby's (link: docs/guide/cache#caching-pages text: page cache) is an efficient means to improve the performance of a website. It keeps a copy of every page rendered and Kirby returns the cached copy for future requests instead of rendering it anew.
+
+Sometimes, it may come handy to bypass the cache for logged-in users.
+
+## Using the page cache
+
+As a built-in default, the page cache is enabled with the `cache.pages` option in `config.php`:
+
+```php "/site/config/config.php"
+return [
+    'cache' => [
+        'pages' => [
+            'active' => true
+        ]
+    ]
+];
+```
+
+Kirby also takes care of emptying the cache whenever content is changed through the panel so that no outdated content is served to users.
```

Comment:
The Panel needs to start with a capital P ;)

Reply:
Fixed.